fairlearn/metrics/_metric_frame.py

```diff
@@ -536,7 +536,7 @@ def difference(
         features, then :attr:`.overall` is multivalued for each metric).
         The result is the absolute maximum of these values.
 
-        Read more in the :ref:`User Guide <control_features_metrics>`.
+        Read more in the :ref:`User Guide <assessment>`.
```

Comment:
Should we point to the assessment page or to the part of the assessment section that introduces the MetricFrame? I'm not sure what page that topic ended up in, though.

Reply:
This makes it match the others. I'd be open to changing it to the precise section, though (would change all of them).

Reply:
Oh ok, no worries. Sorry about that, I do not want to suggest cascading changes. I will delete this suggestion.

Reply:
I'm willing to change all of them - it was only on the previous PR to split up the text into sections that I wanted to minimise changes.